### ISSUE TITLE
Support diff mode

### ISSUE THREAD

**Issue by lu-zero** [ferrous-systems/flamegraph#43]
perl flamegraph has a mode to generate diff graphs, it would be useful to have it.

**Comment by dabreegster** [comment]
https://github.com/jonhoo/inferno/issues/62 is marked done -- I'm curious what's needed here to support diffs?

(Awesome tool by the way, thanks for building it!)

**Comment by djc** [comment]
Yes, inferno supports this. It's not obvious to me that this fits in well with the flamegraph CLI UX, but feel free to make a proposal for how the CLI should look for this and we can discuss it.

**Comment by zz85** [comment]
@dabreegster

it probably wasn't clear from that linked issue, but the code in the [PR](https://github.com/jonhoo/inferno/pull/96/files#diff-3d6ece56946ce9d27f9625ee168146e9f7006f59b514e53f3e90d5ee42f2eedbR25) or even running `inferno-diff-folded --help` explains how to use.

```
  $ inferno-diff-folded folded1 folded2 | inferno-flamegraph > diff2.svg
```

**Comment by gatlinnewhouse** [comment]
using current flamegraph `.svg`s from `cargo flamegraph` with `inferno-diff-folded folded1 folded2 | inferno-flamegraph > diff2.svg` is giving the following error:

```
[ERROR inferno::flamegraph] No stack counts found
Error: Custom { kind: InvalidData, error: "No stack counts found" }
```

I am guessing `--post-process` with something will output a folded stack that can be used as a source (since the `.svg`s do not seem to work)

**Comment by djc** [comment]
flamegraph is only getting passive maintenance right now -- happy to review a PR, but I'm unlikely to work on this myself anytime soon.
